srtcore/core.cpp

```diff
@@ -5529,6 +5544,8 @@ void * srt::CUDT::tsbpd(void* param)
             tsNextDelivery = steady_clock::time_point(); // Ready to read, nothing to wait for.
         }
 
+        SRT_ATR_UNUSED bool bWakeupOnSignal = true;
```

Comment:
Rename to `bWokeUpOnSignal` to describe an event, not the intent.

Reply:
This was intended to be a substantive describing the performed activity: "a wakeup". But I agree this sounds ambiguous.

Reply:
```suggestion
        SRT_ATR_UNUSED bool bWokenupOnSignal = true;
```